analysis/magefrost/src/normalizers/CometStormLinkNormalizer.ts

```diff
@@ -0,0 +1,45 @@
+import SPELLS from 'common/SPELLS';
+import EventLinkNormalizer, { EventLink } from 'parser/core/EventLinkNormalizer';
+import {
+  CastEvent,
+  DamageEvent,
+  EventType,
+  GetRelatedEvents,
+  HasRelatedEvent,
+} from 'parser/core/Events';
+import { Options } from 'parser/core/Module';
+
+const CAST_BUFFER_MS = 2500;
+
+export const FROM_HARDCAST = 'FromHardcast';
+export const HITS_TARGET = 'HitsTarget';
+
+const EVENT_LINKS: EventLink[] = [
+  {
+    linkRelation: FROM_HARDCAST,
+    linkingEventId: SPELLS.COMET_STORM_DAMAGE.id,
+    linkingEventType: EventType.Damage,
+    referencedEventId: SPELLS.COMET_STORM_TALENT.id,
+    referencedEventType: EventType.Cast,
+    forwardBufferMs: CAST_BUFFER_MS,
```

Comment:
Unless I'm mistaken, the cast event will always be before the damage events, meaning `forwardBufferMs` should be zero. Leaving it at 2.5 seconds is going to cause a lot of false positives once the set bonus comes into play.